.envrc.example

```diff
@@ -9,3 +9,6 @@ export ETHERSCAN_API_KEY=''
 
 # Required for forking
 export ALCHEMY_URL=''
+
+#required for mainnet governance forking testnets
+export PathToGovernanceRepo=''
```

Comment:
Let's have this name follow the convention above for constants, which is to have all caps with _ spaces.

Reply:
Also...does this get used anywhere?  I don't see a reference